File: c++/busqLineal.cpp
#include <iostream>
#include <random>
using namespace std;

int main(){

    random_device rd;
    default_random_engine gen(rd());
    uniform_int_distribution<int> dist(1,20);

    int numeros[10];

    for(int i=0;i<10;i++){
        numeros[i] = dist(gen);
    }   

    int numeroBuscado = dist(gen), posic=0;
    bool bandera = false;

    for(int i=0;i<10;i++){
        if (numeros[i] == numeroBuscado){
            posic = i;
            bandera =true;
        }
    }
    
    if (bandera == true){
        cout<<"El numero "<<numeroBuscado<<", fue encontrado en la posicion "<<posic<<" del array"<<endl;
    }
    else{
        cout<<"El numero "<<numeroBuscado<<", no fue encontrado dentro del array"<<endl;
    }

    return 0;
}

File: python/busqLineal.py
import random

array = [1,2,3,4,5,6,7,8,9,10]
numeroBus = random.randint(1,20)
bandera = False
posic = 0

for i in range(len(array)):
    if array[i] == numeroBus:
        bandera = True
        posic = i+1

if bandera == True:
    print(f"El numero {numeroBus}, fue encontrado en la posicion {posic} del array")
else:
    print(f"El numero {numeroBus}, no fue encontrado dentro del array")
